host-configs/CVX/cvx-gpu.cmake

```diff
@@ -0,0 +1,64 @@
+site_name(HOST_NAME)
+set(CONFIG_NAME "cvx-gpu" CACHE PATH "" FORCE)
+message( "CONFIG_NAME=${CONFIG_NAME}" )
+
+set(GCC_ROOT "/util/gcc/gcc-9.3.0" CACHE PATH "")
+set(CMAKE_C_COMPILER "${GCC_ROOT}/bin/gcc" CACHE PATH "" FORCE)
+set(CMAKE_CXX_COMPILER "${GCC_ROOT}/bin/g++" CACHE PATH "" FORCE)
+set(CMAKE_Fortran_COMPILER "${GCC_ROOT}/bin/gfortran" CACHE PATH "" FORCE)
+set(ENABLE_FORTRAN OFF CACHE BOOL "" FORCE)
+
+set(ENABLE_MPI ON CACHE PATH "" FORCE)
+set(MPI_ROOT "/vend/intel/parallel_studio_xe_2018_update3/compilers_and_libraries_2018.3.222/linux/mpi/intel64" CACHE PATH "")
+set(MPI_C_COMPILER "${MPI_ROOT}/bin/mpicc" CACHE PATH "" FORCE)
+set(MPI_CXX_COMPILER "${MPI_ROOT}/bin/mpicxx" CACHE PATH "" FORCE)
+set(MPI_Fortran_COMPILER "${MPI_ROOT}/bin/mpiifort" CACHE PATH "" FORCE)
+set(MPIEXEC "${MPI_ROOT}/bin/mpirun" CACHE PATH "" FORCE)
+set(MPIEXEC_NUMPROC_FLAG "-n" CACHE STRING "")
```

Comment:
This can be moved in a common file, given that the `MPI_ROOT` is already defined.

Reply:
done